app/components/Sidebar/index.tsx

```diff
@@ -233,7 +233,7 @@ const Sidebar = (): JSX.Element => {
         </List>
       </div>
     );
-  }, [adminAccess, germlineAccess, handleSidebarClose, pathname, reportsAccess, sidebarMaximized]);
+  }, [managerAccess, germlineAccess, handleSidebarClose, pathname, reportsAccess, sidebarMaximized]);
```

Comment:
Technically will adminAccess still needed to be a dependency since it was used in line 45?

Reply:
fixed